doc/rtd/topics/instancedata.rst

```diff
@@ -197,6 +248,263 @@ instance:
 
 .. sourcecode:: json
 
+  {
+   "_beta_keys": [
+    "subplatform"
+   ],
+   "availability_zone": "us-east-1b",
+   "base64_encoded_keys": [],
+   "merged_cfg": {
```

Comment:
Just to confirm, this doc update is an example of a instance-data-sensitive.json file as this value is redacted in the world-readable instance-data.json correct?

Reply:
rharper, yes I'll update the the docs to clarify that